gau2grid/c_generator.py

```diff
@@ -267,17 +267,22 @@ def generate_c_gau2grid(max_L,
     gg_header.write("#endif")
     gg_header.write("#endif /* GAU2GRID_GUARD_H */")
 
+    # Create header directory if not present
+    header_path=os.path.join(path,'gau2grid')
```

Comment:
` = ` (space on either side of the equals for consistent formatting).

Reply:
`"` instead of `'`, too